### PR TITLE
feat: add channel section

### PR DESCRIPTION
**Pull request by edouardparis**

### CODE REVIEW

File: content/reference/channel.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+## List channels
+
+List all channels.
+
+The response is [paginated](#pagination).
+
+{{% http method="get" %}}/v1/channels{{% /http %}}
+
+## List user channels
```
Comment (yansal) [inline content/reference/channel.md]: move this endpoint to the end of this section

File: content/reference/channel.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Retrieves the channel with the given ID.
+
+{{% http method="get" %}}/v1/channels/:id{{% /http %}}
+
+| Parameter | Description |
+| --------- | ----------- |
+| `:id`     | Channel ID  |
+
+## List channel selection
```
Comment (yansal) [inline content/reference/channel.md]: what do you think about using the search.projects endpoint for this?

Comment (edouardparis) [reply content/reference/channel.md]: the selection too ? Ok for me if it is possible.

Comment (yansal) [reply content/reference/channel.md]: I don't see what could make it impossible

File: content/reference/channel.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+weight: 22
+---
+
+# Channel
+
+## Channel resource
+
+| Field          | Type                     | Description                                                      |
+| -------------- | ------------------------ | ---------------------------------------------------------------- |
+| `absolute_url` | string                   | Link to the channel page on the Ulule website                    |                                                                                                                                 |
```
Comment (yansal) [inline content/reference/channel.md]: remove trailing `|`

File: content/reference/channel.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+List all channels.
+
+The response is [paginated](#pagination).
+
+{{% http method="get" %}}/v1/channels{{% /http %}}
+
+## Retrieve a channel
+
+Retrieves the channel with the given ID or the given slug.
```
Comment (yansal) [inline content/reference/channel.md]: Let's just document the slug parameter, we never document two path parameters + it's simpler for the user as there is only one way to do it.

File: content/reference/channel.md (hunk 2 above)
Comment (edouardparis) [reply content/reference/channel.md]: what would be the qualifier ? `selected:bool` ?

Comment (yansal) [reply content/reference/channel.md]: After thinking twice:
1. For the automatic list, is there a reason why we couldn't use the search.projects endpoint with the tag_id qualifier?
2. For the selection, it's probably better to add a new endpoint, as you did here. This endpoint should move to the `Project` section, as it returns a list of projects.

Comment (edouardparis) [reply content/reference/channel.md]: > 1. For the automatic list, is there a reason why we couldn't use the search.projects endpoint with the tag_id qualifier?

It think none, it is just to be explicit.

> 2.
ok for me

**Pull request merged**